cmd/utils.go

```diff
@@ -192,27 +201,33 @@ func commonOwnerOrOperatorSetup(ctx context.Context, from string) (agentAddr com
 		return common.Address{}, nil, accounts.Account{}, nil, err
 	}
 
-	as := util.AgentStore()
+	as := util.AccountsStore()
 	ks := util.KeyStore()
 	backends := []accounts.Backend{}
 	backends = append(backends, ks)
 	manager := accounts.NewManager(&accounts.Config{InsecureUnlockAllowed: false}, backends...)
 
-	opEvm, opFevm, err := as.GetAddrs(util.OperatorKey)
+	opEvm, opFevm, err := as.GetAddrs(string(util.OperatorKey))
 	if err != nil {
+		if err == util.ErrKeyNotFound {
+			return common.Address{}, nil, accounts.Account{}, nil, fmt.Errorf("agent accounts not found in wallet. Setup with: glif wallet create-agent-accounts")
+		}
 		return common.Address{}, nil, accounts.Account{}, nil, err
 	}
 
-	owEvm, owFevm, err := as.GetAddrs(util.OwnerKey)
+	owEvm, owFevm, err := as.GetAddrs(string(util.OwnerKey))
 	if err != nil {
+		if err == util.ErrKeyNotFound {
+			return common.Address{}, nil, accounts.Account{}, nil, fmt.Errorf("agent accounts not found in wallet. Setup with: glif wallet create-agent-accounts")
+		}
 		return common.Address{}, nil, accounts.Account{}, nil, err
 	}
 
 	var fromAddress common.Address
 	// if no flag was passed, we just use the operator address by default
-	switch from {
-	case "", opEvm.String(), opFevm.String():
-		funded, err := as.IsFunded(ctx, PoolsSDK, opFevm, util.OperatorKeyFunded, opEvm.String())
+	switch strings.ToLower(from) {
+	case "", opEvm.String(), opFevm.String(), string(util.OperatorKey):
```

Comment:
Does the `strings.ToLower(from)` interfere with potentially checksummed addresses resulting from the `.String()` method?

I.e. if `opEvm.String()` returns checksummed address, then this case will never match?

Reply:
It definitely does. This is a bug.

I'll wrap the `.String()` methods to make them lowercase too.

Reply:
Appears to be fixed.